frontend-v2/src/features/data/dataValidation.ts

```diff
@@ -257,12 +257,20 @@ function validateCatCovariates(state: StepperState) {
   return validationErrors;
 }
 
+export function validateDosingRows(state: StepperState) {
+  const amountField = state.fields.find(
+    (field) => state.normalisedFields.get(field) === "Amount",
+  );
+  return (
+    amountField !== undefined &&
+    !state.data.every((row) => row[amountField] === ".")
```

Comment:
I'm not sure if this should also check for rows where the amount column is an empty string.